README.MD

```diff
@@ -223,6 +223,16 @@ export interface MyModel {
 ```
 
 ### Overriding route template
+
```

Comment:
Is this related to the goal of the PR?

Reply:
It’s only related because I had to make changes that would break everyone who has a custom template. Those people using custom templates should probably be told that they’re at risk. If not, then we have to consider every data change to the templates as a breaking change. I don’t want to do that many major versions because people will miss out on cool features then.